### DISCUSSION TITLE
Valkey support?

### DISCUSSION THREAD

**Discussion by gvolpe** [Grokzen/docker-redis-cluster#166] [Ideas]
Hi, thanks a lot for this project! We've been using it successfully to develop [this Scala client](https://redis4cats.profunktor.dev/). 

Due to the Redis corporate move, we're moving over to [Valkey](https://valkey.io/download/) and was wondering if this is something you have considered adding support for? Or that could consider supporting?

**Comment by Grokzen** [comment]
@gvolpe There is possibly a fork in the process of being built :) I think it is a better place for me to experiment with my build process that i have wanted to change around a ton of things in the build parts that i just have not got around to. Doing a docker-valkey-cluster build might just be the best place for it. No timeline or promises tho

**Reply by gvolpe** [reply]
Awesome to hear, thanks for your response :)
